syntax = "proto3";

package breezedemo;

service BreezeApi {
    rpc Connect (ConnectRequest) returns (ConnectResponse);
    
    rpc GetCustomerDetails (CustomerRequest) returns (ApiResponse);
}

message CustomerRequest {
    string ApiSession = 1;
}

message ApiResponse {
    string Success = 1;
    int32 Status = 2;
    string Error = 3;
}

message ConnectRequest {
    string AppKey = 1;
    string SecretKey = 2;
    string ApiSession = 3;
}

message ConnectResponse {
    string message = 1;
}